jetstream/first-run-easy-as-one-click-ver-1.toml

```diff
@@ -0,0 +1,43 @@
+[experiment]
+start_date = "2022-11-29"
+end_date = "2023-01-03"
+enrollment_period = 15
+
+[metrics]
+weekly = ['days_of_use', 'qualified_cumulative_days_of_use', 'search_count', 'organic_search_count', 'active_hours', 'unenroll',
```

Comment:
You don't need to manually specific most of these since they're defaults. You only need to list the new ones. So this could be replaced with 
```toml
weekly = ['imported_bookmarks', 'imported_logins', 'imported_history']
overall = ['imported_bookmarks', 'imported_logins', 'imported_history']
```